faults.csv

```diff
@@ -1,63 +1,63 @@
+NRP,NIRP or individual SSA interlock,NIRP or other individual SSA interlocks like PPS,"If the realtime status is OK (check legend), reset interlocks. If not, contact NIRP system experts.",SSA,PYDM,$PYDM/nirp/nirp_Lband_diagnostic_graphical.ui,"{""AREA"": {LINAC}, ""CM"": {CRYOMODULE}, ""RFNAME"": ""CM{CRYOMODULE}""}",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:SSA:,NRP_PRMT,1,,2,NIRP or individual SSA interlock
+SSP,SSA Permit,SSA permit to turn on faulted,,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,SSA_LTCH,0,,2,SSA Permit
+SSA,SSA Faulted,SSA not on,,SSA,EDM,$EDM/llrf/rf_srf_ssa_{cm_OR_hl}.edl,,,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:SSA:,FaultSummary.SEVR,,2,2,SSA Faulted
+BCS,BCS LLRF Drive Fault,BCS fault is interrupting LLRF drive (only affects CM01 in practice),Wait for fault to clear,CM,EDM,$EDM/bcs/ops_lcls2_bcs_main.edl,,,ACCL:{LINAC}:{CRYOMODULE}00:,BCSDRVSUM,0,,2,BCS LLRF Drive Fault
+FRO,LLRF Frequency Offset,"LLRF Frequency is offset from nominal because LLRF has shifted the frequency, or in frequency scan mode",Confirm that this isn't being actively used intentionally and then Rack Larger Range Freq Scan > Shift > Stop,RACK,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=1,SELCHAR=2",A,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,FREQSUM,,2,2,
+FRO,LLRF Frequency Offset,"LLRF Frequency is offset from nominal because LLRF has shifted the frequency, or in frequency scan mode",Confirm that this isn't being actively used intentionally and then Rack Larger Range Freq Scan > Shift > Stop,RACK,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=1,SELCHAR=2",B,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,FREQSUM,,2,2,LLRF Frequency Offset
+CSI,Cryo Summary Interlock,"CM Cryo Summary Interlock (He pressure, He Level, Cryo Heartbeat, and CP Ready) faulted","Check if the summary is OK and if yes, reset interlocks",CAV,EDM,$EDM/llrf/rf_srf_intlk_cryo.edl,,,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,CRYO_LTCH,0,,2,Cryo Summary Interlock
+RLK,Resonance/Interlock Link,Resonance/ Interlock Link,,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,RESLINK_LTCH,0,,2,Resonance/Interlock Link
+FPG,FPGA PLL Lock,The LLRF digital clock is no longer guranteed to be locked to the accelerator due to the FPGA detecting a temporary loss of externally supplied LO for the onboard clock,,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,PLL_LTCH,0,,2,FPGA PLL Lock
+IOC,EPICS IOC Watchdog,Interlock trips if the watchdog detects that the IOC is down,Hover over question mark button on screen ,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,IOCWDOG_LTCH,0,,2,EPICS IOC Watchdog
+BLV,Beamline Vacuum,Beamline Vacuum too high,Wait until beamline vacuum is OK and then reset interlocks,RACK,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",A,ACCL:{LINAC}:{CRYOMODULE}00:,BMLNVACA_LTCH,0,,2,

```

Comment:
wouldn't we want to be notified about beamline vacuum incidents? I think for this one it is reasonable to contact the SRF on call person